### DISCUSSION TITLE
Auto Updating Cask

### DISCUSSION THREAD

**Discussion by benr** [Homebrew/discussions#2774] [Writing Formulae/Casks] [answered]
Hello, 

 I'm a little confused by the purpose of LiveChecks.  It indeed checks for the latest version but doesn't appear to actually help users get the latest version.  Are they simply to avoid someone submitting a formula/casks for an old version?

 More to the point... I'm maintaining a cask that produces weekly releases.  Is there any way to avoid my having to PR an update of the cask every week?

 Thanks!

**Comment by carlocab** [comment, marked as answer]
The `livecheck` also helps check whether the version registered in Homebrew/cask is outdated. Try, for example, `brew livecheck foo`.

You can simplify creating a weekly PR by using `brew bump-cask-pr`. It should do pretty much everything for you. You could probably even automate this in CI.

Or, you can skip some weeks. No one's forcing you to do the work. Maybe someone else will update the cask in the meantime.
